docs/source/quickstart.mdx

```diff
@@ -0,0 +1,405 @@
+
+Hello! This tutorial gets you up and running with Apollo Federation. Unlike other articles in this documentation, this tutorial uses some Apollo tools and terminology that are still in **preview**.
+
+**This content is not yet recommended for use in a production environment.** However, we expect these tools to leave preview soon, so this tutorial is a great way to begin learning about federation using upcoming best practices.
+
+> This tutorial requires Node.js version 12 or later. 
+
+## Federation concepts
+
+In a federated architecture, multiple GraphQL APIs are **composed** into a single federated data graph. The individual APIs are called **subgraphs**, and they're composed into a **supergraph**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+Usually, each subgraph corresponds to a different service in your backend. The supergraph is then represented by a **gateway**, which routes each incoming query to the appropriate combination of subgraphs and returns the combined result.
+
+The supergraph's schema is the combination of each subgraph's schema, plus some special federation-specific directives. Each subgraph's schema can even reference and extend types that originate in a _different_ subgraph. (<a href="../entities" target="_blank">Learn more</a>)
+
+This architecture enables clients to query data from multiple services simultaneously, just by querying the gateway.
+
+Now that we have a high-level understanding of federation concepts, let's jump in.
+
+## 1. Install the Rover CLI
+
+**Rover** is Apollo's new CLI for managing data graphs, including federated ones. It's currently in preview.
+
+Install it like so:
+
+```shell
+npm install -g @apollo/rover
+```
+
+After installing, run `rover` in your terminal with no arguments to confirm that it installed successfully. We'll use various Rover commands in later steps.
+
+## 2. Create a gateway
+
+> You can also [clone our example gateway on GitHub](https://github.com/apollographql/supergraph-demo). The GitHub example is **not** identical to the gateway you create in this tutorial, but it demonstrates the same concepts.
+
+As mentioned in the [preface](#preface-federated-architecture), your federated supergraph is represented by a **gateway** that routes queries to various subgraphs. For this tutorial, we'll use some Apollo-hosted example services as our subgraphs, and we'll set up our own gateway in front of them.
+
+With the help of the `@apollo/gateway` library, [Apollo Server](https://www.apollographql.com/docs/apollo-server/) can act as our federated gateway.
+
+In a new Node.js project, first install the required libraries:
```

Comment:
If I'm not a Node developer, this line is confusing to me. What is a new node.js project? How do I start one?